cmd/servegoissues/servegoissues.go

```diff
@@ -76,6 +76,11 @@ func main() {
 		box-shadow: 0 1px 1px rgba(0, 0, 0, .05);
 	}
 </style>`,
+		BodyPre: `
+{{/* Override new comment component to link to original issue for leaving comments. */}}
+{{define "new-comment"}}<div class="event" style="margin-top: 20px; margin-bottom: 100px;">
+	View <a href="https://github.com/golang/go/issues/{{.Issue.ID}}#new_comment_field">original issue</a> to comment.
```

Comment:
This currently hardcodes `golang/go` here. I think it's valid for serve**go**issues and we should merge this as is. However, for https://github.com/bradfitz/go-issue-mirror/issues/7#issuecomment-242991322, we can refactor this to use the `RepoSpec` and generate a valid link to whichever repository/issue tracker it corresponds to.

I'd rather defer doing that until then because it'll be easier, given we'll have a better idea of the requirements and assumptions we can make. Will it support github-only projects? Or others too? That'll be important to know.